desktop/src/main/java/bisq/desktop/main/dao/governance/make/MakeProposalView.java

```diff
@@ -178,19 +205,44 @@ protected void deactivate() {
             makeProposalButton.setOnAction(null);
     }
 
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Bindings, Listeners
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    private void addBindings() {
+        proposalTypeComboBox.managedProperty().bind(isProposalPhase);
+        proposalTypeComboBox.visibleProperty().bind(isProposalPhase);
+        nextProposalBox.managedProperty().bind(isProposalPhase.not());
+        nextProposalBox.visibleProperty().bind(isProposalPhase.not());
+        proposalTitledGroup.textProperty().bind(proposalGroupTitle);
+    }
+
+    private void removeBindings() {
+        proposalTypeComboBox.managedProperty().unbind();
+        proposalTypeComboBox.visibleProperty().unbind();
+        nextProposalBox.managedProperty().unbind();
+        nextProposalBox.visibleProperty().unbind();
+        proposalTitledGroup.textProperty().unbind();
+    }
 
     ///////////////////////////////////////////////////////////////////////////////////////////
     // DaoStateListener
     ///////////////////////////////////////////////////////////////////////////////////////////
 
     @Override
     public void onNewBlockHeight(int height) {
-        boolean isProposalPhase = daoFacade.isInPhaseButNotLastBlock(DaoPhase.Phase.PROPOSAL);
-        proposalTypeComboBox.setDisable(!isProposalPhase);
-        if (!isProposalPhase)
+        isProposalPhase.set(daoFacade.isInPhaseButNotLastBlock(DaoPhase.Phase.PROPOSAL));
+        if (isProposalPhase.get()) {
+            proposalGroupTitle.set(Res.get("dao.proposal.create.selectProposalType"));
+        } else {
+            proposalGroupTitle.set(Res.get("dao.proposal.create.phase.inactive"));
             proposalTypeComboBox.getSelectionModel().clearSelection();
+            updateTimeUntilNextProposalPhase(height);
```

Comment:
Here u get the current cycle so it will show the past proposal phase.
<img width="814" alt="screen shot 2019-02-05 at 15 07 44" src="https://user-images.githubusercontent.com/1449498/52278811-3126d480-2958-11e9-969b-c2a3c75ff133.png">
Either we add the duration of a full cycle, or we show just a diff. text and pointing to the dashboard to see when phases start and end (i think thats better as there its easier to understand if u see all phases).